### ISSUE TITLE
This example with qiskit_machine_learning does not work

### ISSUE THREAD

**Issue by filippotramonto** [MicrosoftDocs/quantum-docs#618]
This example with qiskit_machine_learning does not work.
It gives me the error "No module named 'qiskit_machine_learning'".
I have tried with both instructions
`!pip install qiskit-machine-learning`
and
`!pip install qiskit
!pip install qiskit-machine-learning` 
before running the cells. I have obtained same results.
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: eb056c67-eed8-4e5c-1d83-470909d85f74
* Version Independent ID: eb056c67-eed8-4e5c-1d83-470909d85f74
* Content: [Adapting Qiskit samples - Azure Quantum ](https://learn.microsoft.com/en-us/azure/quantum/how-to-adapt-qiskit)
* Content Source: [articles/how-to-adapt-qiskit.md](https://github.com/MicrosoftDocs/quantum-docs/blob/main/articles/how-to-adapt-qiskit.md)
* Service: **azure-quantum**
* Sub-service: **core**
* GitHub Login: @SoniaLopezBravo
* Microsoft Alias: **sonialopez**

**Comment by SoniaLopezBravo** [comment]
Hi @filippotramonto, thanks for your feedback. I'll try to reproduce your error. 
1. Are you running your code in Azure Quantum notebooks? 
2. Have you copied `!pip install qiskit` and `!pip install qiskit-machine-learning` in different cell blocks, or in the same cell block? Try using different cell blocks and click on **Run all**.

**Comment by filippotramonto** [comment]
Hi Sonia,

1. yes, in the notebooks on the Azure Quantum workspace

2. yes, of course in different cell, executed before the execution of the
cell with the import

I have discovered later in another doc page (
https://learn.microsoft.com/en-us/azure/quantum/overview-quantum-machine-learning)
a comment in the cell with the pip install that says to restart the kernel
after running the cell.

> # Restart the kernel after running that cell
> !pip install qiskit-machine-learning


With this "trick" the example works!

I suggest to add this instruction in the documentation, in both pages.
(Also the latter page, because the comment is in the cell block, not in the
main text. At first moment I did not see the comment in the cell)

Best regards
Filippo Tramonto, PhD


Il giorno gio 3 ago 2023 alle ore 13:13 Sonia Lopez <
***@***.***> ha scritto:
